### ISSUE TITLE
Bulk Data Access

### ISSUE THREAD

**Issue by danieldjewell** [linuxhw/hw-probe#156]
I noticed that there doesn't appear to be a way to do a bulk download of parsed/machine readable data that has been collected.  

While "published" on Github (#122), much of the data is in Markdown files... Which makes it exceedingly difficult to analyze.

Is there a database/JSON/msgpack/Parquet/CSV/[insert easily machine readable format here] collection available? (Preferably a "parsed"/"processed" version that makes it easily searchable.)

**Comment by lee-b** [comment]
Agree, current website and markdown fikes are not very helpful. The raw database should be provided as a regular snapshot so that people can slice and dice the data as needed.
